frontend/src/container/withActivity.jsx

```diff
@@ -182,8 +182,27 @@ const withActivity = (WrappedComponent, setActivityList, setActivityNextPage, re
           }
         }
       }
+      let activityList = props.activity.list
+      if (data.event_type.includes(TLM_ET.CONTENT) && data.event_type.includes(TLM_SUB.COMMENT) && data.event_type.includes(TLM_CET.MODIFIED)) {
```

Comment:
add comment to explain what and why this is done

Reply:
done